tests/admintools.test.js

```diff
@@ -128,23 +153,41 @@ describe('admintools', function ( ) {
 
       var d3 = require('d3');
       //disable all d3 transitions so most of the other code can run with jsdom
-      d3.timer = function mockTimer() { };
+      //d3.timer = function mockTimer() { };
+      let timer = d3.timer(function mockTimer() { });
+      timer.stop();
+      
+      var cookieStorageType = self.localStorage._type
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cookieStorageType' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)